apimocks/datagrid/tiles.json

```diff
@@ -0,0 +1,74 @@
+{
+    "_links": {
+        "self": {
+            "href": "http://husky.lo:7878/admin/api/datagrid/tiles"
+        }
+    },
+    "_embedded": {
+        "items": [
+            {
+                "id": "1",
+                "title": "Folder title 1",
+                "countElements": 100
+            },
+            {
+                "id": "2",
+                "title": "Folder title 2",
+                "countElements": 120
+            },
+            {
+                "id": "3",
+                "title": "Folder title 3",
+                "countElements": 130
+            },
+            {
+                "id": "4",
+                "title": "Folder title 4",
+                "countElements": 140
+            },
+            {
+                "id": "5",
+                "title": "Folder title 5",
+                "countElements": 150
+            },
+            {
+                "id": "6",
+                "title": "Folder title 6",
+                "countElements": 160
+            },
+            {
+                "id": "7",
+                "title": "Folder title 7",
+                "countElements": 170
+            },
+            {
+                "id": "8",
+                "title": "Folder title 8 with a very very long title",
+                "countElements": 180
+            },
+            {
+                "id": "9",
+                "title": "Folder title 9",
+                "countElements": 190
+            },
+            {
+                "id": "10",
+                "title": "Folder title 10",
+                "countElements": 200
+            },
+            {
+                "id": "11",
+                "title": "Folder title 11",
+                "countElements": 210
+            },
+            {
+                "id": "12",
+                "title": "Folder title 12",
+                "countElements": 220
```

Comment:
If you call it like that I would say `mediaCount`. Having `count` first makes it sound like a function, and if you only call it elements I would suspect it to also include sub collections.